webrtc/libjingle/xmpp/boshsocket.cc

```diff
@@ -155,7 +156,15 @@ namespace buzz {
 #ifdef FEATURE_ENABLE_SSL
     if (tls_ != buzz::TLS_DISABLED) {
       first_socket = rtc::SSLAdapter::Create(first_socket);
+// For non-production builds, relax certificate requirements. This will allow the use
+// of a debugging proxy, for example.
+#ifndef _PRODUCT
```

Comment:
I don't think it needs to be with this PR.  I think we just need to change the StagePresence gclient invocation to potentially set this in the webrtc build preprocessor settings if it is set in our CMake config.  Not exactly sure what the steps for that are.